docs/source/tasks/community_contributions/index.md

```diff
@@ -0,0 +1,96 @@
+<!-- THIS IS ALL GENERATED DOCUMENTATION. DO NOT MODIFY THIS FILE -->
+[asset-badge]: https://img.shields.io/badge/download%20asset-yes-blue.svg
+[dense-reward-badge]: https://img.shields.io/badge/dense%20reward-yes-green.svg
+[sparse-reward-badge]: https://img.shields.io/badge/sparse%20reward-yes-green.svg
+[no-dense-reward-badge]: https://img.shields.io/badge/dense%20reward-no-red.svg
+[no-sparse-reward-badge]: https://img.shields.io/badge/sparse%20reward-no-red.svg
+[demos-badge]: https://img.shields.io/badge/demos-yes-green.svg
+# Community Contributed Projects
```

Comment:
we might want to rename this to just External Benchmarks/Tasks

we are including our own work, which has a website with the same theme as maniskill lol. The repo is separate because its cleaner that way.

not necessarily now but we should find a way for community members to upload their own stuff here and have some way verify it, would be nice to gather a long list of resources of people's good work. Can complement the papers list that @NirajPudasaini has helped compile later.

Reply:
oh shoot sorry I didn't notice the request. I've added the changes